lib/manager/pip_setup/index.spec.ts

```diff
@@ -105,7 +108,7 @@ describe(getName(__filename), () => {
       expect(
         await extractPackageFile(
           'raise Exception()',
-          '/tmp/folders/foobar.py',
+          'folders/foobar.py',
```

Comment:
To prevent confusion; otherwise `cwd` would  be `/tmp/github/some/repo/tmp/folders`.